docs/glossary.rst

```diff
@@ -94,6 +95,23 @@ email alert if something seems wrong. Only system admins connect
 to this server, and they may only do so using Tor.
 
 
+Onion Service
+-------------
+
+Tor onion services provide anonymous inbound connections to websites and other servers exclusively over the Tor network. For example, SecureDrop uses onion services for the *Journalist Interface* and *Source Interface* websites, as well as for adminsitrative access to the servers in SSH-over-Tor mode.
```

Comment:
adminsitrative-> administrative

Reply:
It may be useful to cross-link Tor's own glossary entry here, which may also offer some inspiration for making the language more accessible:
https://support.torproject.org/onionservices/

Reply:
@eloquence I noticed that Tor's glossary doesn't apply the same [caution when linking onion addresses](https://github.com/freedomofpress/securedrop-docs/pull/128#discussion_r555407001).

> For example, the DuckDuckGo onion is https://... (link)

Because of that, I added the cross-reference in a separate commit that you can keep, squash or remove when performing the rebase.